### PR TITLE
Update README for GitHub Actions

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

This just updates the readme to include GitHub Actions status and CodeCov report links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,8 @@
...
...anch=kinetic-devel)](https://github.com/ros-planning/moveit/actions/workflows/format.yml?branch=kinetic-devel) [![BuildAndTest](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml/badge.svg?branch=kinetic-devel)](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml?branch=kinetic-devel) | [![Format](https://github.com/ros-planning/moveit/actions/workflows/format.yml/badge.svg?branch=melodic-devel)](https://github.com/ros-planning/moveit/actions/workflows/format.yml?branch=melodic-devel) [![BuildAndTest](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml/badge.svg?branch=melodic-devel)](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml?branch=melodic-devel) | [![Format](https://github.com/ros-planning/moveit/actions/workflows/format.yml/badge.svg?branch=master)](https://github.com/ros-planning/moveit/actions/workflows/format.yml?branch=master) [![BuildAndTest](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml/badge.svg?branch=master)](https://github.com/ros-planning/moveit/actions/workflows/industrial_ci_action.yml?branch=master) |
```
Comment (rhaschke) [inline README.md:25]: You could add the Noetic build for the master column?
https://build.ros.org/job/Ndev__moveit__ubuntu_focal_amd64/

**Review by rhaschke — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tylerjw) [reply README.md:25]: done!

**Review by rhaschke — APPROVED** [review]

**Comment by rhaschke** [comment]
We should definitely do something about the failing buildfarm...

**Comment by tylerjw** [comment]
> We should definitely do something about the failing buildfarm...

Looks like it's failing for the same reason as the ament_lint test:

```
KeyError: 'moveit_resources_prbt_moveit_config'
```

We should probably do a release of moveit_resources and that would fix this.

**Pull request merged**